superset/views/core.py

```diff
@@ -364,20 +364,11 @@ def pre_delete(self, obj):
     @expose('/add', methods=['GET', 'POST'])
     @has_access
     def add(self):
-        widget = self._add()
-        if not widget:
-            return redirect(self.get_redirect())
-
-        sources = ConnectorRegistry.sources
-        for source in sources:
-            ds = db.session.query(ConnectorRegistry.sources[source]).first()
-            if ds is not None:
-                url = "/{}/list/".format(ds.baselink)
-                msg = _("Click on a {} link to create a Slice".format(source))
-                break
-
-        redirect_url = "/r/msg/?url={}&msg={}".format(url, msg)
-        return redirect(redirect_url)
+        flash(__(
+            "To create a new slice, you can open a data source "
+            "through the `Sources` menu, or alter an existing slice "
+            "from the `Slices` menu"), "danger")
```

Comment:
should we use a danger flash here, or an info flash? info flash makes more sense to me.